guides/source/active_job_basics.md

```diff
@@ -165,6 +165,7 @@ Here is a noncomprehensive list of documentation:
 - [Sneakers](https://github.com/jondot/sneakers/wiki/How-To:-Rails-Background-Jobs-with-ActiveJob)
 - [Sucker Punch](https://github.com/brandonhilkert/sucker_punch#active-job)
 - [Queue Classic](https://github.com/QueueClassic/queue_classic#active-job)
+- [Delayed Job](https://github.com/collectiveidea/delayed_job)
```

Comment:
I would opt to link to `https://github.com/collectiveidea/delayed_job#rails-42`, albeit a little dated and recursive. We could also try updating the title to `Rails 4.2+` in the DJ readme.